README.md

```diff
@@ -1,78 +1,92 @@
-# PlayTak-API 
+# PlayTak 
+
+This is the new monorepo for the PlayTak services 
 
 ## Description
 
 ![Unit Tests](https://github.com/USTakAssociation/playtak-api/actions/workflows/ci.yml/badge.svg)
 
 
-This is the Play Tak API project which contains multiple endpoints for servicing data to the Play Tak UI
+This is repo contains the api and Tak server for servicing data to the Play Tak UI client and games ui
```

Comment:
Typo: "This is repo contains..."

Reply:
Also, is it "Play Tak" or "PlayTak"?

Reply:
both? changed it to PlayTak for consistency